chrome_extension_starfish_ide_from_scratch/content.js

```diff
@@ -0,0 +1,57 @@
+
+current_url=window.location.href;
+var page_uri ="ide://#"+window.location.href;
+ 
+if(current_url.indexOf("https://github.com/")!=-1)
```

Comment:
a matter of style but why not keep '{' close to the if ?

Reply:
sure

Reply:
> a matter of style but why not keep '{' close to the if ?

fixed on latest [commit](https://github.com/maxandersen/starfish/pull/1/#discussion_r407161132)